src/shared/tabs/index.ts

```diff
@@ -1,33 +1,36 @@
 import { LitElement, html, css, PropertyValueMap } from "lit";
 import { customElement, property, state, queryAssignedElements } from "lit/decorators.js";
+import "./tab";
 
 export const tag = "bdo-tabs";
 
 @customElement(tag)
 export class BdoTabs extends LitElement {
   @property({ type: Number }) selectedIndex = 0;
+  @property({ type: String }) label!: string;
   @state() private _tabTitles: string[] = [];
   @queryAssignedElements({ selector: "bdo-tab" }) private _tabs!: HTMLElement[];
 
   override render() {
     return html`
-      <div class="tabs-header" role="tablist" aria-label="Tab Panel">
+      <div class="tabs--list" role="tablist" .ariaLabel="${this.label}">
         ${this._tabTitles.map(
           (title, index) => html`
-            <button
-              class="tab-title ${index === this.selectedIndex ? "selected" : ""}"
+            <a
+              href="#tabpanel-${index}"
```

Comment:
De href maakt het tabpanel niet zichtbaar en de href is niet uniek wanneer er meerdere tabs op een pagina staan.

Reply:
De `[aria-controls]` verwijst ook naar dezelfde `id` als de anker link. En id's behoren uniek te zijn. Er moeten dus unieke tabpanel ids komen. Een `button` of `a` element gebruiken maakt het verschil niet.
Nu is de vraag of er tabpanels aangeleverd moeten worden met unieke titels en/of ids, of dat dit component er voor moet zorgen dat elk id uniek is. Eigenlijk kan dat laatste niet echt als je ook leesbare url's/id's wil hebben.